spec/controllers/forgot_passwords_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+describe ForgotPasswordsController do 
+
+  describe "POST create" do 
+
+    before do
+      @alice = Fabricate(:user)
```

Comment:
You should use `let` rather than instance variables. Read more [here](http://betterspecs.org/#let).